mcs/build/profiles/basic.make

```diff
@@ -9,6 +9,8 @@ use_monolite := $(wildcard $(monolite_flag))
 
 MONOLITE_MSCORLIB = $(monolite_path)/mscorlib.dll
 
+PLATFORMS = darwin linux win32
```

Comment:
This is not how you want to fix it, look at https://github.com/mono/mono/compare/master...luhenry:corefx-System.Net.NetworkInformation#diff-8a17f52e0062026d1a9555a3bae175e8R298 for the fix.

Reply:
That link doesn't work.

Reply:
[This link](https://github.com/mono/mono/compare/master...luhenry:corefx-System.Net.NetworkInformation#diff-8a17f52e0062026d1a9555a3bae175e8R298) should work